scrapyd_k8s/api.py

```diff
@@ -14,6 +14,15 @@
 launcher = (config.launcher_cls())(config)
 scrapyd_config = config.scrapyd()
 
+# middleware that adds "node_name" to each response if it is a JSON
+@app.after_request
+def after_request(response: Response):
+    if response.is_json:
+        data = response.json
+        data["node_name"] = config.scrapyd().get("node_name", launcher.get_node_name())
+        response.data = jsonify(data).data
+    return response
+
```

Comment:
Let's move this more near the end of the file, after the routes, because it is more like an extra thing to add, not the main thing.